Library/Homebrew/manpages/brew.1.md.erb

```diff
@@ -104,6 +104,10 @@ can take several different forms:
     `https://raw.github.com/Homebrew/homebrew-core/master/Formula/git.rb`.
     The formula file will be cached for later use.
 
+  * A bottle:
```

Comment:
I'd see this as an arbitrary URL as well. Maybe more examples in that block would be a better option?

Reply:
This can't be used for every command that accepts a formula but just for install. This could be added to the documentation for `install` if desired.

Reply:
@SMillerDev The example and text clearly show that the URL should point to a formula `.rb` file. Moreover, a local file is accepted, too -- which is _not_ a URL.

Since [a local formula file is also accepted](https://stackoverflow.com/questions/50196723/caching-homebrew-bottles-and-formulas-on-travis-ci/50257889#comment93204476_50257889), I guess "arbitrary URL" could be changed to "arbitrary file or URL", and added that "It could point to either a formula file or a bottle."

Reply:
@MikeMcQuaid I specifically checked that other commands that accept a formula specification (at least, `info`, `deps` and `uninstall`) also accept a bottle. They seem to be using the formula file embedded into the bottle ('cuz the commands with the bottle file keep working even if I rename the formula file in the tap directory).

Reply:
@SMillerDev https://github.com/native-api/brew/pull/1

Reply:
> 
> 
> Maybe it can be renamed to URI?

[A local path is not a URI, either](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Definition) ;-)

Reply:
> 
> 
> @SMillerDev [native-api#1](https://github.com/native-api/brew/pull/1)

In case this was unclear: this is a PR to this PR. Other than making it manually (against my branch), you can make it by going to the "files changed" tab, pressing the pencil button above a file, making and saving the changes, then choosing "suggest changes" in the form that would appear below. Others can then suggest changes to your changes. and so on. The recently-added "suggest changes" feature doesn't yet allow recursiveness.

Reply:
@native-api Cool, thanks for checking.